editions/tw5.com/tiddlers/variables/Behaviour of variables invoked via widget attributes.tid

```diff
@@ -8,4 +8,4 @@ type: text/vnd.tiddlywiki
 |!how declared|!behaviour|
 |\define|Textual substitution of parameters is performed on the body text. No further processing takes place. The result after textual substitution is used as the attribute's value|
 |<<.wlink SetWidget>>, <<.wlink LetWidget>>, <<.wlink VarsWidget>>, \procedure, \widget|Body text is retrieved as-is and used as the attribute's value.|
-|\function|When a function is invoked as `<div class=<<macro>>/>`, it is a synonym for `<div class={{{[function[macro]]}}}/>`. As with any filtered transclusion (i.e. triple curly braces), all results except the first are discarded. That first result is used as the attribute's value. Note that functions are recursively processed even when invoked in this form. In other words a filter expression in a function can invoke another function and the processing will continue|
\ No newline at end of file
+|\function|When a function (e.g. `myfun`) is invoked as `<div class=<<myfun>>/>`, it is a synonym for `<div class={{{[function[myfun]]}}}/>`. As with any filtered transclusion (i.e. triple curly braces), all results except the first are discarded. That first result is used as the attribute's value. Note that functions are recursively processed even when invoked in this form. In other words a filter expression in a function can invoke another function and the processing will continue|
```

Comment:
For functions to be also usable in filter strings, they need to have a "dot" in the name. eg: `my.function` or `f.myfun`

Reply:
That's true, but not in this case. Do you recommend to always include the dot in the name?
Of course, for consistency I agree to use dot in function name everywhere in the doc.

Reply:
I think we should go with "best practice". The problem I found when doing more advanced stuff -- is that "renaming" everything without issues is relatively hard. So having "good" function names in the first place is worth the time spent.

Reply:
okay. I add the dot in the function name for this PR.